lib/services/ios-project-service.ts

```diff
@@ -347,7 +347,14 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				if(firstPostInstallIndex !== -1 && firstPostInstallIndex !== projectPodfileContent.lastIndexOf(IOSProjectService.PODFILE_POST_INSTALL_SECTION_NAME)) {
 					this.$logger.warn(`Podfile contains more than one post_install sections. You need to open ${this.projectPodFilePath} file and manually resolve this issue.`);
 				}
-
+				
+				let pbxprojFilePath = path.join(this.platformData.projectRoot, this.$projectData.projectName + IOSProjectService.XCODE_PROJECT_EXT_NAME, "xcuserdata");
+				if(!this.$fs.exists(pbxprojFilePath).wait()) {
+					this.$logger.info("Creating project scheme...");
+					let createScheme_rb = `echo \"require 'xcodeproj'; xcproj = Xcodeproj::Project.open('${this.$projectData.projectName}.xcodeproj'); xcproj.recreate_user_schemes; xcproj.save\" | ruby`;
```

Comment:
The ruby gem `cocoapods` depends on `xcodeproj` so it is automatically installed. Do you have the latest `cocoapods v.0.38.2` ? Users can manually uninstall gem `xcodeproj`. However doing so will cause a few lines down the execution of `pod install` command to fail as well although there would be installed `cocoapods` (with missing `xcodeproj`). Here is a snippet of the error message:
`/Library/Ruby/Site/2.0.0/rubygems/dependency.rb:315:in ``to_specs': Could not find 'xcodeproj' (~> 0.26.3) among 35 total gem(s) (Gem::LoadError)``

Perhaps there should be a check for installed `xcodeproj` gem as well. I've added it in the latest commit.